spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe RegisteredApplicationsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:my_app) { create(:application, user: user) }
+
+  describe "GET index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it "assigns my_app to @registered_applications" do
+      get :index
+      expect(assigns(:registered_applications)).to eq([my_app])
+    end
+  end
+
+  describe "GET show" do
+    it "returns http success" do
+      get :show, {id: my_app.id}
+      expect(response).to have_http_status(:success)
+    end
+
+    it "renders the #show view" do
+      get :show, {id: my_app.id}
+      expect(response).to render_template :show
+    end
+
+    it "assigns my_app to @registered_application" do
+      get :show, {id: my_app.id}
+      expect(assigns(:registered_application)).to eq(my_app)
+    end
+  end
+
+  describe "GET new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+
+    it "renders the #new view" do
```

Comment:
I would not write these tests either except for the status one above.

Reply:
So the tests were because nothing was really working so I was trying to work from the basics, but than the tests weren't working anyway, so that's why they are some what incomplete.